.github/workflows/publish.yml

```diff
@@ -16,6 +16,25 @@ jobs:
           w3up-proof: ${{ secrets.W3UP_PROOF }}
           w3name-private-key: ${{ secrets.W3NAME_PRIVATE_KEY }}
           w3name-revision: ${{ secrets.W3NAME_REVISION }}
+      - if: failure()
+        uses: slackapi/slack-github-action@v2.0.0
+        with:
+          method: chat.postMessage
+          token: ${{ secrets.SLACK_BOT_TOKEN }}
+          payload: |
+            {
+              "channel": "alerts",
+              "text": "Publishing `${{ github.event.repository.name }}` failed",
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
```

Comment:
Is `mrkdwn` correct? It looks like a typo to me, I would expect `markdown` instead.

Reply:
Yes, I copy/pasted this from spark-evaluate: https://github.com/CheckerNetwork/spark-evaluate/blob/main/.github/workflows/ci.yml#L122. See also https://api.slack.com/reference/surfaces/formatting#basic-formatting